app/releasemanager.cpp

```diff
@@ -20,7 +20,9 @@
 #include "releasemanager.h"
 #include "drivemanager.h"
 
-#include "isomd5/libcheckisomd5.h"
+extern "C" {
+#include <libcheckisomd5.h>
```

Comment:
Yes and I did have a PR that did that (https://github.com/rhinstaller/isomd5sum/pull/9) but when I noticed that this is not general practice I did cancel the PR.
It's not the C libraries fault that you can overload functions in C++.